src/main.rs

```diff
@@ -1,18 +1,19 @@
 mod config;
 use actix_web::{get, App, HttpServer, Responder};
 use anyhow::Result as AnyhowResult;
+use tracing_actix_web::TracingLogger;
 
 #[get("/")]
```

Comment:
So this looks like a GET endpoint, right?
if so, is this `root` method called when this `/` endpoint is hit?